eazy/version.py

```diff
@@ -1,5 +1,5 @@
-# git describe --tags
-__version__ = "0.5.2.dev45"
-__long_version__ = "0.5.2-45-g4cff093"
-__version_hash__ = "g4cff093"
-    
\ No newline at end of file
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+__version__ = version = '0.5.3.dev70+g3ecc085.d20221121'
+__version_tuple__ = version_tuple = (0, 5, 3, 'dev70', 'g3ecc085.d20221121')
```

Comment:
You should add `eazy/version.py` to .gitignore.  Always best to not have this under version control.